English translation (#71)
personium/new-docs: src/getting-started/README.md

## Introduction

> This document is an introductory article for those who are new to Personium. The purpose of this document is to familiarize you with the basics of Personium so that you can use it.

Reviewer: ```suggestion
This Getting Started guide is for those who are new to Personium. It will cover the fundamental and technical knowledge of Personium.
```

> ## Prerequisite knowledge

Reviewer: ```suggestion
## Prerequisite  
```

> If you have not read [Personium architecture](../user_guide/001_Personium_Architecture.md) as a common prerequisite knowledge of Personium, it is recommended to read it.

Reviewer: ```suggestion
It is recommended to read the [Personium architecture](../user_guide/001_Personium_Architecture.md) chapter before you continue further.
```

> ## Chapter structure

Reviewer: ```suggestion
## Chapter's layout
```

> The Getting Started is roughly divided into a chapter "1. Preparing Personium" for Personium operators and a chapter "2. App Development" for app developers. The position of each chapter is as shown in the figure below.

Reviewer: ```suggestion
This Getting Started guide is roughly divided into two chapters . "1. Preparing Personium" for Personium operators and "2. Devloping app" for app developers. Each chapter corresponds to the layers as shown in the figure below.
```

![Getting Started 2 steps](assets/users-for-getting-started.png)

> The details of the chapter structure are as follows.

Reviewer: ```suggestion
The following is the layout of each chapter.
```

> 1. Preparation of Personium (for Personium operators)

Reviewer: ```suggestion
1. Preparing Personium (for Personium operators)
```

    1. [Setup Personium Unit](./setup-unit.md)
    2. [Payout of Cell](../unit-administrator/tutorial.md)
    3. [Building a sample app](./setup-sample-apps.md)

> 2. App development (for app developers)

Reviewer: ```suggestion
2. Developing app (for app developers)
```

    1. [Introduction](./appdev-introduction.md)
    2. [Management Tool (Unit Manager)](./appdev-management-tool.md)
    3. [Flow of application development](./appdev-process.md)
    4. [App implementation](./appdev-impl.md)

> > If you can use the built-in Personium, you can skip the chapter "1. Preparing Personium" and read from the chapter "2. App Development".

Reviewer: ```suggestion
> If you have access to a Personium Unit, you can skip "1. Preparing Personium" and start from "2. Developing app".
```
